docs/thirdparty.rst

```diff
@@ -37,7 +37,20 @@ Third Party Applications
 .. content list start
 
 
-.. replace this line with the first entry
+LiveStreamLinkGUI
+---------------------
+
+.. image:: https://raw.githubusercontent.com/techmouse/livestreamlinkgui/master/screenshot.png
+    :alt: LiveStreamLinkGUI
+
+:Description: A lightweight BASH script
+:Type: Graphical User Interface
+:OS: |Linux| |MacOS| And any other OS that supports BASH, Zenity, and typical \*nix commands
+:Author: `Mouse <https://github.com/techmouse>`_
+:Website: https://github.com/techmouse/livestreamlinkgui
```

Comment:
I think the website link should be removed if it's the same as the source code URL, as it is redundant information. As I said, the field list doesn't have to be strict. 😃